src/role-access/containers/exclusion/exclusion-home/exclusion-home.component.ts

```diff
@@ -15,7 +15,7 @@ import * as fromFeature from '../../../store';
 import { CheckAnswersComponent } from '../check-answers/check-answers.component';
 import { ChooseExclusionComponent } from '../choose-exclusion/choose-exclusion.component';
 import { ChoosePersonRoleComponent } from '../choose-person-role/choose-person-role.component';
-import { DescribeExclusionComponent } from '../describe-exclusion/describe-exclusion.component';
+import { DescribeExclusionContainerComponent } from '../describe-exclusion/describe-exclusion.component';
```

Comment:
if this is a container could we name it just DescribeExclusionContainer, the sufix *Component seems redundant and misleading.

Reply:
Done